app/models/practice_editor_session.rb

```diff
@@ -0,0 +1,129 @@
+class PracticeEditorSession < ApplicationRecord
+  belongs_to :practice
```

Comment:
we also probably want to add a relationship to the PracticeEditorSession and the PracticeEditor model

Reply:
? I don't believe there is a relationship between these models.